docs/reference/customize_admin.rst

```diff
@@ -0,0 +1,65 @@
+Customize admin
+================
+
+There are few admins in sonata page bundle that can be customize using `Admin Extension`_
+As example if you want to add a new field in ``configureListFields``, You can do like this
+
+::
+
+    namespace App\Admin\Extension;
+
+    use Sonata\AdminBundle\Admin\AbstractAdminExtension;
+    use Sonata\AdminBundle\Datagrid\ListMapper;
+    use Symfony\Component\DependencyInjection\Attribute\AutoconfigureTag;
+
+    #[AutoconfigureTag(name: 'sonata.admin.extension', attributes: ['target' => 'sonata.page.admin.page'])]
+    final class PageAdminExtension extends AbstractAdminExtension
+    {
+        public function configureListFields(ListMapper $list): void
+        {
+            $list->add('id');
+        }
+    }
+
+And you will see the field in the end of the table
+
+.. image:: ../images/customize_admin__add_id.png
+
+And for order the field to the begin of your table can use the method ``reorder``
```

Comment:
There is no occurence of `reorder` in the SonataAdmin doc
https://docs.sonata-project.org/projects/SonataAdminBundle/en/4.x/search/?q=reorder&check_keywords=yes&area=default

I think it would be better to document this in this doc.

Not sure it's better in the list section https://docs.sonata-project.org/projects/SonataAdminBundle/en/4.x/reference/action_list/#
 in the Extension section  https://docs.sonata-project.org/projects/SonataAdminBundle/en/4.x/reference/extensions/
or in both

Reply:
I guess it will be used more for ExtensionAdmin!

Reply:
I think too

Reply:
I think it would be better to
- Improve SonataAdmin doc first
- Eventually add a doc like yours in SonataPage with link to SonataAdmin then :)

Reply:
Ok I'll Open a PR to Admin bundle :)